app/frontend/src/locales/es/translation.json

```diff
@@ -0,0 +1,131 @@
+{
+    "pageTitle": "Azure OpenAI + AI Search",
```

Comment:
Thanks for the starter translations! Were these AI generated? Should I get a native speaker to review them?